Resources/Markdown/faq.md

```diff
@@ -117,24 +119,35 @@ Push an updated `Package.swift` and the Swift Package Index will automatically u
 
 ---
 
-<h3 id="incompatible-license">Why do some packages show a red license lozenge?</h3>
+<h3 id="incompatible-license">Why do some packages show red license information?</h3>
 
-If a package's license shows in a red lozenge rather than a green one, it is for one of two reasons:
+If a package's license shows with a background that isn't green, it is for one of three reasons:
 
-1. We're unable to detect software license the package uses.
+1. The package does not have a license.
+2. We're unable to detect software license the package uses.
```

Comment:
_the_ software license

I can't make direct edits in the review, can I? I thought there existed something like that.